packages/client/mutations/UpdatePokerScopeMutation.ts

```diff
@@ -294,7 +294,7 @@ const UpdatePokerScopeMutation: StandardMutation<TUpdatePokerScopeMutation, Hand
       const searchQuery = getSearchQueryFromMeeting(meeting, service)
       if (!searchQuery) return
       const {searchQueryString, searchQueryFilters} = searchQuery
-      SendClientSegmentEventMutation(atmosphere, 'Updated Poker Scope', {
+      SendClientSideEvent(atmosphere, 'Updated Poker Scope', {
```

Comment:
+1 what's your thinking around removing `Mutation` from the name here? It's still a mutation, and as all of the other mutation names end with `Mutation`, I think we should do that here too